packages/beacon-node/src/network/metadata.ts

```diff
@@ -35,16 +35,15 @@ export type MetadataModules = {
  */
 export class MetadataController {
   private onSetValue: (key: string, value: Uint8Array) => void;
-  private config: BeaconConfig;
+  private networkConfig: NetworkConfig;
   private _metadata: fulu.Metadata;
 
   constructor(opts: MetadataOpts, modules: MetadataModules) {
-    this.config = modules.config;
+    this.networkConfig = modules.networkConfig;
     this.onSetValue = modules.onSetValue;
     this._metadata = opts.metadata ?? {
       ...ssz.fulu.Metadata.defaultValue(),
-      // TODO: @matthewkeil check if this needs to be updated for custody groups
-      cgc: Math.max(this.config.CUSTODY_REQUIREMENT, this.config.NODE_CUSTODY_REQUIREMENT),
+      cgc: modules.networkConfig.custodyConfig.advertisedCustodyGroupCount,
```

Comment:
Looks like `custodyConfig` is private.  Should we use the getter?

```suggestion
      cgc: modules.networkConfig.getCustodyConfig().advertisedCustodyGroupCount,
```